REFERENCE.md

```diff
@@ -2201,7 +2203,10 @@ Struct[{
     Optional['OOMPolicy']                 => Enum['continue', 'stop','kill'],
     Optional['OOMScoreAdjust']            => Integer[-1000,1000],
     Optional['Environment']               => String,
-    Optional['EnvironmentFile']           => Variant[Stdlib::Unixpath,Pattern[/-\/.*/]],
+    Optional['EnvironmentFile']           => Variant[
+      Stdlib::Unixpath,Pattern[/-\/.+/],
+      Array[Variant[Stdlib::Unixpath,Pattern[/-\/.+/]],1],
```

Comment:
Does this allow for the special value `~`?

Reply:
I wasn't aware that `~` was used in `EnvironmentFile`. https://www.freedesktop.org/software/systemd/man/systemd.service.html doesn't mentioned it.

Reply:
Whoops, nevermind, got this confused with `WorkingDirectory` from #321.